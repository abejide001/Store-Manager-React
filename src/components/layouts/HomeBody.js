import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../../assets/images/logo.png';
import '../../assets/css/style.css';
import Header from './Header';
import Footer from './Footer';

export default function HomeBody() {
  return (
    <div className="landing-page">
      <Header />
      <section id="showcase">
        <div className="container">
          <h1>Store Manager.</h1>
          <p>
Manage your sales and product inventory records.
          </p>
        </div>
      </section>
      <Footer />
    </div>
  );
}
