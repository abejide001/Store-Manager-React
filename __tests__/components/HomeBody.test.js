/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomeBody from '../../src/components/layouts/HomeBody';

describe('<HomeBody />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<HomeBody />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
