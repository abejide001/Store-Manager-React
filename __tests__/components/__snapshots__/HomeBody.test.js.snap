// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeBody /> render() renders the component 1`] = `
<div>
  <section
    id="showcase"
  >
    <div
      className="container"
    >
      <h1>
        Store Manager.
      </h1>
      <p>
        Manage your sales and product inventory records.
      </p>
    </div>
  </section>
</div>
`;
